src/controller/sig.py

```
@@ -69,11 +74,11 @@ async def create_sig_ctrl(
         session.commit()
     except IntegrityError:
         session.rollback()
-        raise HTTPException(409, detail="시그장 자동 가입 중 중복 오류가 발생했습니다")
+        raise HTTPException(409, detail="피그장 자동 가입 중 중복 오류가 발생했습니다")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)